### ISSUE TITLE
Driver not working under windows7

### ISSUE THREAD

**Issue by miloshev** [pothosware/SoapyFCDPP#5]
In windows7 Funcube Pro plus dongle is recognized as an audio device.
Using Zadig does not help either.
Any ideas ?

**Comment by guruofquality** [comment]
I think this is a linux only project because of libHID for control and libalsa for the streaming. And zadig is primarily for libusb (as far as I understand) which FCDPP does not use. How did you compile support on windows, was it cygwin or something?

**Comment by ast** [comment]
Yes I only wrote this for Linux and tested on raspberry pi. It probably
isn't that difficult to extend to work in Windows but I don't have time or
knowledge for this.

Josh, I remember you suggested a cross platform audio API, which one was
it? I might give that a go if I find some time for it.

**Comment by guruofquality** [comment]
Not sure about libhid, but for audio, there is a header only rt-audio project that gives an API to a bunch of different audio frameworks: https://github.com/pothosware/SoapyAudio/tree/master/RtAudio

And that said, if SoapyAudio has no problem using the FCDPP stream, then you could just instantiate and call into SoapyAudio directly from SoapyFCDPP.

I ended up using portaudio for PothosAudio blocks which worked quite well as a cross platform audio API. Although its not header only :-) https://github.com/pothosware/PothosAudio

**Comment by ast** [comment]
I will check it out. Should work well as long as there's not a lot of
unnecessary conversations..
